Week-1/MUSIC_PLAYER/MUSIC.css

```diff
@@ -0,0 +1,96 @@
+*{
+    font-family: 'Lato';
+    color:black;
+  }
+
+  .main_content ,table, footer{
+    position: relative;
+    left:20px;
+    top:-10px;
```

Comment:
Ideally we should not be using top with relative position. In case if you want to then please use transform property here.